### PR TITLE
Update grow

### PR DESCRIPTION
**Pull request by xtpkt**
This follows along with PR #2920, bringing the beauty of ash plants to tShock.

### CODE REVIEW

**Review by RenderBr — DISMISSED** [review]
seems fucking epic to me

**Review by ghost — APPROVED** [review]

**Review by ghost — DISMISSED** [review]
looks good to me, approved.

**Review by RenderBr — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]
Looks good to me

**Review by muuuted — APPROVED** [review]

**Review by muuuted — DISMISSED** [review]

**Review by numanumaa — DISMISSED** [review]
Written and behaves as expected.

File: docs/changelog.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ Use past tense when adding new entries; sign your name off when you add or chang
 * Allowed multiple test cases to be in TShock's test suite. (@drunderscore)
 * Fixed unable to use Purification/Evil Powder in jungle. (@sgkoishi)
 * Set the `GetDataHandledEventArgs.Player` property for the `SyncTilePicking` data handler. (@drunderscore)
+* Added Ash tree to `/grow` (@xtpkt)
```
Comment (hakusaro) [inline docs/changelog.md]:
```suggestion
* Added Ash tree to `/grow`. (@xtpkt)
```

**Review by hakusaro — COMMENTED** [review]
Everything else is probably fine

**Review by hakusaro — APPROVED** [review]
seems legit? I can't wait to find out what I've forgotten about this

**Review by QuiCM — APPROVED** [review]

**Pull request closed without merging**